### PR TITLE
add 8 segment to display emulator

### PR DESCRIPTION
**Pull request by borgdog**
just a quick change to enable 8 segment virtual display

### CODE REVIEW

File: mpf/config_spec.yaml [hunk 1]
```diff
@@ -479,7 +479,6 @@ drop_targets:
     enable_keep_up_events: event_handler|event_handler:ms|None
     disable_keep_up_events: event_handler|event_handler:ms|None
     ball_search_order: single|int|100
-    max_reset_attempts: single|int|None
```
Comment (jabdoa2) [inline mpf/config_spec.yaml]: Not sure if you wanted to remove this.

File: mpf/config_spec.yaml [hunk 2]
```diff
@@ -488,7 +487,6 @@ drop_target_banks:
     __valid_in__: machine, mode
     __type__: device
     drop_targets: list|machine(drop_targets)|
-    max_reset_attempts: single|int|None
```
Comment (jabdoa2) [inline mpf/config_spec.yaml]: Same here.

File: mpf/config_spec.yaml [hunk 3]
```diff
@@ -1019,7 +1017,6 @@ multiball_locks:
     priority: single|int|1
     reset_all_counts_events: event_handler|event_handler:ms|None
     reset_count_for_current_player_events: event_handler|event_handler:ms|None
-    empty_lock_devices_on_ball_end: single|bool|false
```
Comment (jabdoa2) [inline mpf/config_spec.yaml]: And here.

**Review by jabdoa2 — CHANGES_REQUESTED** [review]
Can you revert the other config_spec changes?

**Comment by borgdog** [comment]
hmmm, I don't think I made those changes.. .I'll see if I can figure out
how to do that.

**Comment by borgdog** [comment]
I checked the box for allow edits from maintainers, but I do not know how to edit or revert the changes not wanted.  Sorry, very new to github and not a developer.

**Comment by jabdoa2** [comment]
> I checked the box for allow edits from maintainers, but I do not know how to edit or revert the changes not wanted. Sorry, very new to github and not a developer.

Dont worry. Will have a look later

**Review by jabdoa2 — APPROVED** [review]

**Pull request merged**